recipes/flask-turbolinks/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "flask-turbolinks" %}
+{% set version = "0.2.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "9086fa5d425db3530efe7eb96d90067f0531158864f0e7130846e534069d6381" %}
+
+package:
+  version: '{{ version }}'
+  name: '{{ name|lower }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.python.org/packages/4d/f3/e7b4ff1fa5c121e81480bf54dcfb8ce61ccab1153cc6d3a00c36f1e7d2a6/Flask-Turbolinks-0.2.0.tar.gz
```

Comment:
Also, can you change the `url` to the canonical form found here:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

I think you can also drop the `fn` since it's in the `url` for a python package like this.

Reply:
The problem was a 404 (not found) error, when having the URL in the canonical form. I don't know why this happened, but it seems that flask-turboflask is not available via the canonical URL.

Reply:
The following link works:

https://pypi.io/packages/source/F/Flask-Turbolinks/Flask-Turbolinks-0.2.0.tar.gz

You just need to change `{% set name = "flask-turbolinks" %}` to `{% set name = "Flask-Turbolinks" %}`

Reply:
Dang it. Didn't know that case matters... Thanks a lot!